glaciers/glaciers.ipynb

```diff
@@ -119,102 +85,44 @@
+    "def clear_selections(event):\n",
+    "    ls.selection_expr = None\n",
     "\n",
-    "## Define plotting functions\n",
+    "clear_button = pn.widgets.Button(name='Clear selection', align='center')\n",
     "\n",
-    "The above plots are useful for understanding the properties of all glaciers worldwide, but what's more interesting is to consider how some particular subset of the glaciers relates to the rest.  To explore this, let's capture the above commands into some functions that will accept a dataset and return viewable plots for that particular data.  That way we can plot selected subsets of the data and compare them to the plots of the full dataset."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "def geo(data):   return hv.Points(data).options(alpha=1)\n",
-    "def elev(data):  return data.to(hv.Scatter, 'mean_elev', 'latdeg', [])\n",
-    "def temp(data):  return data.hist('avg_temp_at_mean_elev', **temp_kw).options(**temp_opts)\n",
-    "def prcp(data):  return data.hist('avg_prcp',              **prcp_kw).options(**prcp_opts)"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "If called with the full dataset:\n",
+    "clear_button.param.watch(clear_selections, 'clicks');\n",
     "\n",
-    "    (geo_bg*rasterize(geo(data), **geo_kw).options(**geo_opts) + datashade(elev(data), **elev_kw).options(**elev_opts) + temp(data) + prcp(data)).cols(2)\n",
+    "total_area = df.area_km2.sum()\n",
     "\n",
-    "these functions will return static plots just like those above. Let's capture that output as a set of low-opacity (`alpha<0.5`) plots to use as a background on which to show selected subsets of the data:"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "static_geo  = rasterize(geo(data),   **geo_kw).options(alpha=0.1, tools=['hover'], active_tools=['box_select'], **geo_opts)\n",
-    "static_elev = datashade(elev(data), **elev_kw).options(alpha=0.1, active_tools=['box_select'], **elev_opts)\n",
-    "static_temp = temp(data).options(alpha=0.1)\n",
-    "static_prcp = prcp(data).options(alpha=0.1)"
+    "def count(data):\n",
+    "    selected_area  = np.sum(data['area_km2'])\n",
+    "    selected_percentage = selected_area / total_area * 100\n",
+    "    return f'## Glaciers selected: {len(data)} | Area: {selected_area:.0f} km² ({selected_percentage:.1f}%)</font>'\n",
+    "\n",
+    "dynamic_count = pn.bind(count, ls.selection_param(df))\n",
+    "\n",
+    "pn.Row(\n",
+    "    pn.pane.Markdown(pn.bind(count, ls.selection_param(df)), align='center', width=600),\n",
+    "    clear_button\n",
+    ").servable(area='header', title='OGGM Glaciers')"
```

Comment:
Panel offers two ways to declare a template. Explicit via instantiation and global via declaration through `.servable(...)`. We opted for the former, so we can remove that.

```suggestion
    ")"
```